gate_functions.lua

```diff
@@ -292,7 +293,7 @@ local get_door_layout = function(pos, facedir, player)
 					local newpos = rotate_pos_displaced(door_node.pos, origin, axis, direction)
 					local newnode = minetest.get_node(newpos)
 					local newdef = minetest.registered_nodes[newnode.name]
-					if not newdef.buildable_to then
+					if not (newdef and newdef.buildable_to) then
```

Comment:
You could use `get_buildable_to` here as well for consistency. `newnode` and `newdef` are not used below from what I can tell.

Reply:
Thanks :-)